stable/joomla/README.md

```diff
@@ -0,0 +1,97 @@
+# Joomla!
+
+[Joomla!](http://www.joomla.org/) is a PHP content management system (CMS) for publishing web content. It includes features such as page caching, RSS feeds, printable versions of pages, news flashes, blogs, search, and support for language international.
+
+## TL;DR;
+
+```console
+$ helm install stable/joomla
```

Comment:
Just realized it.. this naming is so weird IMO.

This would fail.

```bash
git clone https://github.com/kubernetes/charts
cd charts
helm install stable/joomla
```

while this would work

```bash
cd ~
helm install stable/joomla
```

success is dependent on the CWD.